README.md

```diff
@@ -20,6 +20,15 @@ The client comes pre-installed on all Ubuntu systems in the debian packages
 images will also contain `ubuntu-advantage-pro` which automates machine attach
 on custom AWS and Azure images.
 
+Additionally, there are 3 PPAs with different release channels of the Ubuntu Advantage Client:
+
+1. Stable: This contains stable builds only.
+    - add with `sudo add-apt-repository ppa:ua-client/stable`
+2. Staging: This contains builds that are being prepared for release to stable.
+    - add with `sudo add-apt-repository ppa:ua-client/staging`
+3. Daily: This PPA is updated every day with the latest changes.
+    - add with `sudo add-apt-repository ppa:ua-client/daily`
+
```

Comment:
First of all: is this the gist of what we want to add for these and is it in an appropriate place?

I made a best guess for these descriptions based on the names. Please let me know if anything is wrong.